frontend/src/scenes/ingestion/IngestionWizard.scss

```diff
@@ -1,9 +1,5 @@
 @import '~/vars.scss';
 
-.background {
```

Comment:
FYI reverting this change, as this is how it's reflected on [the design](https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=3865%3A26823). Granted, I still need to update the rest of the UI 🙈